src/styles/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  border: 1px solid var(--gray-elements);
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+  background-color: var(--white);
+  justify-content: center;
+}
```

Comment:
Consider using SCSS nesting to group related styles together. For example, you can nest the styles for `.card__img`, `.card__name`, etc., within the `.card` block for better organization.